src/react/ui-elements/Veeam/VeeamConfiguration.tsx

```diff
@@ -56,6 +60,24 @@ const Configuration = () => {
   };
   const formRef = useRef(null);
   const { next } = useStepper(VeeamStepsIndexes.Configuration, VEEAM_STEPS);
+  const xCoreConfig = useXcoreConfig('run');
+  const { useClusterCapacity } = useXCoreLibrary();
+  const { clusterCapacity, clusterCapacityStatus } =
+    useClusterCapacity(xCoreConfig);
+
+  useEffect(() => {
+    if (clusterCapacityStatus === 'success') {
+      const capacityValue = prettyBytes(parseInt(clusterCapacity, 10), {
+        locale: 'en',
+        binary: true,
+      }).split(' ')[0];
+      const capacityUnitValue = prettyBytes(parseInt(clusterCapacity, 10), {
+        locale: 'en',
+      }).split(' ')[1];
```

Comment:
We should also use the binary format here, and use the iB notation in the form IMO